ccd-definition/CaseEvent.json

```diff
@@ -242,5 +242,27 @@
     "PreConditionState(s)": "Submitted",
     "PostConditionState": "Submitted",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "Shared_Storage_DRAFTType",
+    "ID": "uploadCourtBundle",
```

Comment:
Please change event ID and description. Since every document can be uploaded there an event shouldn't be called `uploadCourtBundle` anymore. Something like `uploadDocumentsAfterSubmission` is better.